src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Tcp/ClientCredentialTypeTests.cs

```diff
@@ -167,9 +167,6 @@ public static void SameBinding_SecurityModeTransport_ClientCredentialTypeCertifi
     }
 
     [ConditionalFact(nameof(Root_Certificate_Installed), nameof(Client_Certificate_Installed))]
```

Comment:
When you run this in ToF (which is of course necessary to verify the fix), you will have to temporarily make this [Fact], because ToF does not understand [ConditionalFact] ad would otherwise skip this test.